### PR TITLE
Hello world Android Setup

### PR DESCRIPTION
**Pull request by minoring**
Navigation Drawer Activity provided by android studio default. #29

### CODE REVIEW

File: Frontend/app/src/main/java/io/happybugs/happybugs/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        } else if (id == R.id.nav_slideshow) {
+
+        } else if (id == R.id.nav_manage) {
+
+        } else if (id == R.id.nav_share) {
+
+        } else if (id == R.id.nav_send) {
+
+        }
```
Comment (shberry) [inline Frontend/app/src/main/java/io/happybugs/happybugs/MainActivity.java]: Should we have 'else' handler too?

File: Frontend/app/src/main/java/io/happybugs/happybugs/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        // Handle action bar item clicks here. The action bar will
+        // automatically handle clicks on the Home/Up button, so long
+        // as you specify a parent activity in AndroidManifest.xml.
+        int id = item.getItemId();
+
+        //noinspection SimplifiableIfStatement
+        if (id == R.id.action_settings) {
+            return true;
+        }
+
```
Comment (shberry) [inline Frontend/app/src/main/java/io/happybugs/happybugs/MainActivity.java]: remove extra line?

File: Frontend/app/src/main/res/values-v21/styles.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<resources>
+
```
Comment (shberry) [inline Frontend/app/src/main/res/values-v21/styles.xml]: remove extra line?

File: Frontend/app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<resources>
+
```
Comment (shberry) [inline Frontend/app/src/main/res/values/styles.xml]: remove extra line?

File: Frontend/.idea/codeStyles/Project.xml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
[the remark this answers is not in the document]
Comment (minoring) [reply Frontend/.idea/codeStyles/Project.xml]: Yes it is Intellij IDE specific files.
I'll remove from git

[the remark this answers is not in the document]
Comment (shberry) [reply Frontend/.idea/codeStyles/Project.xml]: Feel free to resolve this when it's done.

[the remark this answers is not in the document]
Comment (shberry) [reply Frontend/.idea/codeStyles/Project.xml]: (I see two more .idea files below. Can you take a look at them too?)

**Comment by shberry** [comment]
Approved. Let's get this merged whenever you're ready.

**Pull request merged**